packages/fuel-gauge/src/transaction-response.test.ts

```diff
@@ -294,11 +294,13 @@ describe('TransactionResponse', () => {
         genesisWallet.address,
         await genesisWallet.signTransaction(request)
       );
+      const submit = await provider.sendTransaction(request);
+
+      const txResponse = new TransactionResponse(submit.id, provider);
```

Comment:
This test was previously identical to the `submitAndAwaitStatus` one above. I have updated it to now test the `statusChange` subscription properly.